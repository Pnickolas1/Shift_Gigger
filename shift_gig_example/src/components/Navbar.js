import React, {Component} from 'react';

class Navbar extends Component {
    render(){
        return (
            <nav className="navbar navbar-expand-lg navbar-light">
                <a className="navbar-brand" href={"/"}>Shift Gig Message Board</a>
                <ul className="navbar-nav mr-auto">
                    <li className="nav-item">
                        <a className="nav-link" href={"/"}>All Posts</a>
                    </li>
                </ul>
            </nav>
        )
    }

}

export default Navbar