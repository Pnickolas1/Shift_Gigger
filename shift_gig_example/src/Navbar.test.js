import React from 'react';
import Navbar from './components/Navbar'
import renderer from 'react-test-renderer';

it('Jest - Navbar snapshop renders correctly', () => {
  const tree = renderer
    .create(<Navbar/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

