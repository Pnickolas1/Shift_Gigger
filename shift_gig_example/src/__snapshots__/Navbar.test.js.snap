// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest - Navbar snapshop renders correctly 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-light"
>
  <a
    className="navbar-brand"
    href="/"
  >
    Shift Gig Message Board
  </a>
  <ul
    className="navbar-nav mr-auto"
  >
    <li
      className="nav-item"
    >
      <a
        className="nav-link"
        href="/"
      >
        All Posts
      </a>
    </li>
  </ul>
</nav>
`;
